docs/get-started.md

```diff
@@ -46,6 +48,18 @@ NOTE: Make sure to check out the appropriate version. We recommend that you chec
     kubectl get deployments --namespace=nginx-example
     ```
 
+#### Minikube configuration
+
+To get Minikube to connect to Minio storage, you'll need to make some modifications to the configurations.
+
+1. In `examples/minio/00-minio-deployment.yaml`, change the value of `spec.type` in the `Service` YAML from `ClusterIP` to `NodePort`.
+
+1. Run `minikube service minio --namespace=heptio-ark --url` to get the value of the Minio URL.
```

Comment:
I would split `Set up server`'s Step 1 into some separate steps:

- apply prereqs
- apply minio deployment
- get minio URL (this will be NODE_URL_OR_IP:NODE_PORT, where NODE_URL_OR_IP can be gotten in the way described here if running on minikube, and NODE_PORT will be generated by Kubernetes after the previous step -- we can add a `kubectl` command to get it)
- update `05-ark-backupstoragelocation.yaml` to set `s3Url` to the minio URL from the previous step
- apply `20-ark-deployment.yaml`, `30-restic-daemonset.yaml`

Then the `Minikube configuration` section can go away entirely, since per previous comment it's not really minikube-specific AND it's always going to be required if running minio in-cluster.

Reply:
this all makes much more sense than what I had before -- it did seem to me that some of the content was not minikube specific but without testing I wasn't sure what.

I have a bit of kubectl for getting the nodeport, but if I'm understanding correctly to get the full URL not on minikube you have to concatenate values? I could use some help with the right commands/script to make this easier on users.

Reply:
yeah, so if you're not on minikube you need to get the external IP/DNS name of the node that the minio pod is running on.  Not sure that we'll be able to provide any kind of generic command for that - we may just have to describe it.

Reply:
I can help out with relevant commands, though - if you leave placeholders I can pull some things together.